vocs.config.ts

```diff
@@ -188,24 +196,19 @@ export default defineConfig({
         },
       ],
     },
-  ],
-  socials: [
-    {
-      icon: "github",
-      link: "https://github.com/scaffold-eth/scaffold-eth-2",
-    },
-    {
-      icon: "github",
-      link: "https://github.com/scaffold-eth/se2-docs",
-      label: "Docs Repo",
-    },
-  ],
-  topNav: [
     {
-      text: "Documentation",
-      link: "/",
-      match: "/",
-    },
+      text: "Links",
+      items: [
+        {
+          text: "Scaffold-ETH 2 GitHub",
+          link: "https://github.com/scaffold-eth/scaffold-eth-2",
+        },
+        {
+          text: "Scaffold-ETH 2 Docs GitHub",
+          link: "https://github.com/scaffold-eth/se2-docs",
+        }
+      ]
+    }
```

Comment:
Ohh do we need this? 

Coz we already have those links in footer? 

<img width="801" height="292" alt="Image" src="https://github.com/user-attachments/assets/f18f9b0c-2974-45af-bb82-b02c1befa382" />